go.mod

```diff
@@ -5,8 +5,8 @@ go 1.13
 require (
 	github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
-	github.com/containers/buildah v1.20.0
-	github.com/containers/common v0.35.3
+	github.com/containers/buildah v1.20.1-0.20210331202609-d29b04dba7fa
```

Comment:
As @nalind pointed out in slack, we now have a v1.20.1 tag upstream.

Reply:
part of next push